policyengine_canada/parameters/gov/provinces/qc/tax/income/rate.yaml

```diff
@@ -0,0 +1,43 @@
+description: Quebec taxes income at these rates, depending on taxable income.
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-CAD
+  rate_unit: /1
+  label: Quebec income tax rates 
+  reference:
+    - title: Revenu Quebec - Income tax rate
```

Comment:
Can you find a legislative reference?

Reply:
The specific income tax rates for individuals are not set out in the text of the legislative Taxation Act (Quebec) , but typically published in the "Income Tax Rate" and on the website of Revenue Quebec.